deploy/charts/alluxio-operator/crds/k8s-operator.alluxio.com_alluxioclusters.yaml

```diff
@@ -97,32 +107,29 @@ spec:
                     type: object
```

Comment:
so for all these etcd values, do we have explicitly requirement to override ourselves?

Reply:
No. If user doesn't override, we will use our helm chart default value. If our helm chart doesn't have default, we use bitnami chart default value.

Reply:
By the way this file is auto-generated by tools too.